features/clipboard-unsanitized-formats.yml.dist

```diff
@@ -4,6 +4,4 @@
 status:
   baseline: false
   support:
-    chrome: "122"
```

Comment:
The cause of this is here:

https://github.com/mdn/browser-compat-data/blob/ebb58c142ff3f1dcade8857de96d16fb8e2f80f1/api/Clipboard.json#L46-L71

The notes don't actually say what isn't supported...

Reply:
Hmm, yeah, that's iffy data. Two thoughts here:

- The partial seems to be the requirement to have the (non-standard?) `clipboard-read` permission. I can't find that permission specified anywhere, which makes it hard to understand.

- Those media types should be subfeatures of `read` instead of notes. Probably something like `text/plain` and `image/png` ought to be in `compute_from` list?